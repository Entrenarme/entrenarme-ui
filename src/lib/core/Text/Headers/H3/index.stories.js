// @flow
import * as React from 'react';
import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';

import H3 from './index';

const ElementDecoration = storyFn => (
  <div style={{ paddingLeft: '15px' }}>{storyFn()}</div>
);

storiesOf('Text/Headers/H3', module)
  .addDecorator(ElementDecoration)
  .add('default', withInfo('Heading 3')(() => <H3>Heading 3</H3>));

storiesOf('Text/Headers/H3', module)
  .addDecorator(ElementDecoration)
  .add('small', withInfo('Heading 3')(() => <H3 s>Heading 3</H3>));

storiesOf('Text/Headers/H3', module)
  .addDecorator(ElementDecoration)
  .add('large', withInfo('Heading 3')(() => <H3 l>Heading 3</H3>));

storiesOf('Text/Headers/H3', module)
  .addDecorator(ElementDecoration)
  .add(
    'with color prop',
    withInfo('Heading 3')(() => <H3 color="red">Heading 3</H3>),
  );
