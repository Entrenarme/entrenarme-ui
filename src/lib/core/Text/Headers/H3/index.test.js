import * as React from 'react';
import { render } from 'react-testing-library';
import 'react-testing-library/cleanup-after-each';

import H3 from './index';

test('renders "Heading" with no props', () => {
  const { container } = render(<H3>Heading</H3>);
  expect(container.firstChild).toMatchSnapshot();
});
