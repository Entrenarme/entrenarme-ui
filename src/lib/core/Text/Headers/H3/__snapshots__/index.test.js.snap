// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders "Heading" with no props 1`] = `
<h3
  class="sc-bdVaJa iHajuG"
>
  Heading
</h3>
`;
